CMakeLists.txt

```diff
@@ -310,12 +315,13 @@ endif()
 #if(APPLE)
 #    add_library(SDL2main STATIC src/SDLmain/macosx/SDLMain.m)
 #    set_source_files_properties(src/SDLmain/macosx/SDLMain.m PROPERTIES LANGUAGE C)
-#elseif(WIN32)
-#    add_library(SDL2main STATIC src/SDLmain/win32/SDL_win32_main.c)
+if(WIN32)
+    add_library(SDL2main STATIC src/SDLmain/windows/SDL_windows_main.c)
+    target_link_libraries(SDL2main PRIVATE shell32)
```

Comment:
This was needed because of the following missing symbol link error targeting arm32 using VC:
```
SDL2main.lib(SDL_windows_main.c.obj) : error LNK2019: unresolved external symbol __imp_CommandLineToArgvW referenced in function main_getcmdline
```